build.sh

```diff
@@ -1,6 +1,7 @@
+#!/usr/bin/env bash
 # Copyright (c) 2018 Dan Petre
 
 # The MIT License (MIT)
 
 python  setup.py bdist_wheel
-# python3 setup.py bdist_wheel
+python3 setup.py bdist_wheel
```

Comment:
I remember when I left commented code in (see my comment about rules above).
I don't think we need this line. It will just run the pip packaging twice and overwrite the package built by the line above it.
Remove.

Reply:
after you remove the python3 line build the package and manually test that the package loads nice in both python 2.7 and 3.5.

later we'll add automated tests.